tabs/tabs-on-bottom.css

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
While testing this style, I noticed two things:

- It should be split in a separate macOS specific style
- For me (on Linux), it somehow puts the bookmarks toolbar above the navigation toolbar. I have no idea what the cause of this could be :/ Happens on both release and Nightly with Photon